### DISCUSSION TITLE
Formatting the date correctly

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#1500] [Help]
Hi I am using a PostgreSQL database via the external connections tab. I really would like to use the built in "Date Picker" under forms in BB. However I can not correctly format the date in the PostgreSQL database I either get [object object] or its null. Am I doing something wrong or is this a formatting issue on the BB side?  

Thank you for any help you can give me.

**Comment by mjashanks** [comment]
Hi Andrew. Thanks for reporting this. I have been trying this myself and cannot get it to work. the best thing I can do is log an issue: https://github.com/Budibase/budibase/issues/1504

We will try and get it fixed asap

**Reply by KernOil** [reply]
Thank you I appreciate it!
